lib/shared/config/merge-configs.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var copyProps = require('copy-props');
+var path = require('path');
+
+function mergeConfigs(env) {
+  var cfg = {};
+  if (env.configFiles.project) {
+    copyConfig(env.config.project, cfg, env.configFiles.project);
+  }
+  if (env.configFiles.userHome) {
+    copyConfig(env.config.userHome, cfg, env.configFiles.userHome);
+  }
```

Comment:
I missed this until I read the test. I believe the `project` config overrides everything in `userNome` config.

Reply:
I'll modify to use `project` config.